lib/model/survey_detail_model.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:equatable/equatable.dart';
+import 'package:survey_flutter_ic/api/response/survey_detail_response.dart';
+import 'package:survey_flutter_ic/model/survey_question_model.dart';
+
+class SurveyDetailModel extends Equatable {
+  final String id;
+  final List<SurveyQuestionModel> questions;
+
+  const SurveyDetailModel({
+    required this.id,
+    required this.questions,
+  });
+
+  @override
+  List<Object?> get props => [questions];
```

Comment:
@Wadeewee we should put all variables to `props` 🙋 
```suggestion
  List<Object?> get props => [id, questions];
```

Reply:
Good catch @manh-t 👍